src/common/options.cc

```diff
@@ -1360,6 +1360,15 @@ std::vector<Option> get_global_options() {
     .set_description("do not automatically mark OSDs 'out' if an entire subtree of this size is down")
     .add_see_also("mon_osd_down_out_interval"),
 
+    Option("mon_custom_subtree_down_out_type", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("")
+    .set_description("the subtree type down out interval independently timing"),
+
+    Option("mon_custom_subtree_down_out_interval", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(7200)
```

Comment:
If we're not going to enable this by default (sensible), I think we should also require any admins who want it to set the interval up from a default of 0?

Reply:
yeah, make sense to me.